### PR TITLE
Feart: Added Home Page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feart: Added Home Page
Home Page:-
1.I can see a landing page where I can see a list of featured categories.
2.Quizzes by Category
I can play multiple
Screenshot from 2022-02-20 21-25-49
 quizzes by category.

### CODE REVIEW

**Review by shaksham08 — APPROVED** [review]
Looks Good To me

File: Home.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+.fa-unlock{
+    position: relative;
+    top: -14rem;
+    margin-left: -14rem;
+    color:var(--white-color);
+    cursor: pointer;
+    font-size:3rem;
+    
+}
+.fa-lock{
```
Comment (fahd23) [inline Home.css:59]: I can't understand the class naming what fa-lock and unlock means here

**Review by fahd23 — APPROVED** [review]

File: Home.css (hunk 1 above)
Comment (neetukumari4858) [reply Home.css:59]: These are the icon's classess .

Comment (fahd23) [reply Home.css:59]: Ohh ok,Fine then

**Pull request merged**